Documentation/ceph-cluster-crd.md

```diff
@@ -134,8 +134,9 @@ The following storage selection settings are specific to Ceph and do not apply t
 - `walSizeMB`:  The size in MB of a bluestore write ahead log (WAL). Include quotes around the size.
 - `journalSizeMB`:  The size in MB of a filestore journal. Include quotes around the size.
 - `osdsPerDevice`**: The number of OSDs to create on each device. High performance devices such as NVMe can handle running multiple OSDs. If desired, this can be overridden for each node and each device.
+- `encryptedDevice`**: Encrypt OSD volumes using dmcrypt ("true" or "false"). By default this option is disabled.
```

Comment:
The `**` still seems to be out of place. Shouldn't it be at the end of line 136?

Reply:
can you explain a bit more? my impression was that ** was a footnote reference like it is for osdsPerDevice

Reply:
agreed, it looks correct to me as a footnote reference

Reply:
The `**` in Markdown terms makes the text bold in the GitHub code preview but not in the actual file rendered preview.